Setup.iss

```diff
@@ -27,17 +27,17 @@
 #define MyAppPublisherURL "https://www.microsoft.com"
 #define MyAppURL "https://github.com/Microsoft/Git-Credential-Manager-for-Windows"
 #define MyAppExeName "git-credential-manager.exe"
-#define Git4WinVer "2.12.0"
+#define Git4WinVer "2.12.2.2"
```

Comment:
@dscho is this correct version of Git for Windows to bundle with the GCM installer (not that many install the GCM directly - since nearly everyone gets the GCM from GfW installations these days 😄 ).